pkg/gds/emails/client.go

```diff
@@ -118,6 +120,32 @@ func (m *EmailManager) SendVerifyContacts(vasp *pb.VASP) (sent int, err error) {
 // SendReviewRequest is a shortcut for iComply verification in which we simply send
 // an email to the TRISA admins and have them manually verify registrations.
 func (m *EmailManager) SendReviewRequest(vasp *pb.VASP) (sent int, err error) {
+	// Create the template context with the admin verification token
+	ctx := ReviewRequestData{
+		VID:                 vasp.Id,
+		RegisteredDirectory: m.conf.DirectoryID,
+	}
+	if ctx.Token, err = models.GetAdminVerificationToken(vasp); err != nil {
+		return 0, err
+	}
+
+	// Remove sensitive data so it's not sent in the form.
+	clone := proto.Clone(vasp).(*pb.VASP)
+	models.SetAdminVerificationToken(clone, "[REDACTED]")
+
+	var contacts = []*pb.Contact{
+		clone.Contacts.Technical, clone.Contacts.Administrative,
+		clone.Contacts.Billing, clone.Contacts.Legal,
+	}
+
+	for _, contact := range contacts {
+		if contact != nil {
+			_, verified, _ := models.GetContactVerification(contact)
+			models.SetContactVerification(contact, "[REDACTED]", verified)
+		}
+	}
```

Comment:
This was very difficult to implement without making changes to the VASP. I ended up finding the `proto.Clone` function and am just relying on it here - I've created tests to protect from regressions in the future - but boy, I can't express how tough it was to not include some data in the email because of all the pointers.